docs/installation.md

```diff
@@ -0,0 +1,251 @@
+
+### 5. Run a restartable docker image
+To run a restartable instance in the background (feel free to place your
+configuration and database files wherever it feels comfortable on your
+filesystem).
+
+**5.1. Move your config file and database**
+```bash
+mkdir ~/.freqtrade
+mv config.json ~/.freqtrade
+mv tradesv3.sqlite ~/.freqtrade
+```
+
+**5.2. Run the docker image**
+```bash
+docker run -d \
+  --name freqtrade \
+  -v ~/.freqtrade/config.json:/freqtrade/config.json \
+  -v ~/.freqtrade/tradesv3.sqlite:/freqtrade/tradesv3.sqlite \
+  freqtrade
+```
+If you are using `dry_run=True` it's not necessary to mount 
+`tradesv3.sqlite`, but you can mount `tradesv3.dryrun.sqlite` if you
+plan to use the dry run mode with the param `--dry-run-db`.
+
+
+### 6. Monitor your Docker instance
+You can then use the following commands to monitor and manage your container:
+
+```bash
+docker logs freqtrade
+docker logs -f freqtrade
+docker restart freqtrade
+docker stop freqtrade
+docker start freqtrade
+```
+
+You do not need to rebuild the image for configuration changes, it will 
+suffice to edit `config.json` and restart the container.
+
+
+# Linux / MacOS
+## 1. Requirements
+Click each one for install guide:
+- [Python 3.6.x](http://docs.python-guide.org/en/latest/starting/installation/), 
+note the bot was not tested on Python >= 3.7.x
+- [pip](https://pip.pypa.io/en/stable/installing/)
+- [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+- [virtualenv](https://virtualenv.pypa.io/en/stable/installation/) (Recommended)
+- [TA-Lib](https://mrjbq7.github.io/ta-lib/install.html)
+
+## 2. First install required packages
+This bot require Python 3.6 and TA-LIB
+
+### 2.1 Linux - Ubuntu 16.04
+
+**2.1.1. Install Python 3.6, Git, and wget**
+```bash
+sudo add-apt-repository ppa:jonathonf/python-3.6
+sudo apt-get update
+sudo apt-get install python3.6 python3.6-venv build-essential autoconf libtool pkg-config make wget git
+```
+
+**2.1.2. Install TA-LIB**  
+Official webpage: https://mrjbq7.github.io/ta-lib/install.html
+```
+wget http://prdownloads.sourceforge.net/ta-lib/ta-lib-0.4.0-src.tar.gz
+tar xvzf ta-lib-0.4.0-src.tar.gz
+cd ta-lib
+./configure --prefix=/usr
+make
+make install
+cd ..
+rm -rf ./ta-lib*
+```
+
+**2.1.3. [Optional] Install MongoDB**  
+Install MongoDB if you plan to optimize your strategy with Hyperopt.
+
+```bash
+sudo apt-get install mongodb-org
+```
+Complete tutorial on [Digital Ocean: How to Install MongoDB on Ubuntu 16.04](https://www.digitalocean.com/community/tutorials/how-to-install-mongodb-on-ubuntu-16-04)
+
+### 2.2. Linux - Other distro
+If you are on a different Linux OS you maybe have to adapt things like:
+
+- package manager (for example yum instead of apt-get)
+- package names
+
+### 2.3. MacOS installation
+
+**2.3.1. Install git and wget**
+```bash
+brew install wget git
+```
+
+**2.3.2. Install Python 3.6**  
+- [Python 3.6 MacOS offical page](https://www.python.org/downloads/mac-osx/)
```

Comment:
IIRC on osx you can just `brew install python3` to get the latest released version of python

Reply:
Yep, that is the preferable method.

Reply:
Fixed